server/consumer.go

```diff
@@ -928,7 +930,7 @@ func (o *Consumer) isFilteredMatch(subj string) bool {
 // Get next available message from underlying store.
 // Is partition aware and redeliver aware.
 // Lock should be held.
-func (o *Consumer) getNextMsg() (string, []byte, uint64, uint64, int64, error) {
+func (o *Consumer) getNextMsg() (subj string, msg []byte, seq uint64, dcount uint64, ts int64, err error) {
```

Comment:
yah, agree, otoh very difficult to use these methods with this being the only way go gives you to document what the outputs are :( having to read a method code to know whats being returned is pretty grim (general go gripe, nothing to do with your code)

Reply:
We could add to the comments above the method/func?

I am not strictly against them but have gotten burnt alot by local aliasing so try to avoid if I can.

Reply:
we could, but thats not really the go way so IDEs dont do much with that information, and its all unformatted mess, we can just remove them i guess

Reply:
No we can keep, just need to be really careful. The bugs manifest in weird ways..